### ISSUE TITLE
World of Horror (913740)

### ISSUE THREAD

**Issue by hakobi0** [ValveSoftware/Proton#4075]
# Compatibility Report
 World of Horror
 913740

## System Information
https://gist.github.com/hakobi0/b212ecfc799dd7bd40d1b060baddbe3b

- Proton version: 5.0-9

## I confirm:
- [ X] that I haven't found an existing compatibility report for this game.

Log: 

> Proton: 1591828072 proton-5.0-9
> SteamGameId: 913740
> Command: ['/home/tempest/.local/share/Steam/steamapps/common/WOH/worldofhorror.exe']
> Options: {'forcelgadd'}
> 

## Symptoms 
When trying to launch the game the "Game is launching" windows briefly opens but the game never does, it doesn't even generate a proper proton log because of this, I'm unsure of the cause of this, other games open just fine, Hotline Miami for example.

Are there any missing dependencies or something?
Thanks

## Reproduction
In Linux Mint, try to open World of Horror.

**Comment by kisak-valve** [comment]
Hello @hakobi0, in general, Linux Mint 18 is based on Ubuntu 16.04, which is older than the support cutoff for Proton (https://github.com/ValveSoftware/Proton/wiki/Requirements) and that's most likely the primary contributor to the issue you're seeing.

**Comment by TheDiscordian** [comment]
I don't observe any issues on Proton 5.0-9.

```
Operating System Version:
    "Manjaro Linux" (64 bit)
    Kernel Name:  Linux
    Kernel Version:  5.6.15-1-MANJARO
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12008000
    X Window Manager:  KWin
    Steam Runtime Version:  steam-runtime_0.20200604.0

Video Card:
    Driver:  X.Org AMD Radeon (TM) RX 480 Graphics (POLARIS10, DRM 3.36.0, 5.6.15-1-MANJARO, LLVM 10.0.0)
    Driver Version:  4.6 (Compatibility Profile) Mesa 20.0.7
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x1002
    DeviceID:  0x67df
    Revision Not Detected
    Number of Monitors:  2
    Number of Logical Video Cards:  1
    Primary Display Resolution:  2560 x 1440
    Desktop Resolution: 4240 x 1777
    Primary Display Size: 62.99" x 35.43" (72.24" diag)
                                            160.0cm x 90.0cm (183.5cm diag)
    Primary VRAM: 8192 MB
```

**Comment by hakobi0** [comment]
> 
> 
> Hello @hakobi0, in general, Linux Mint 18 is based on Ubuntu 16.04, which is older than the support cutoff for Proton (https://github.com/ValveSoftware/Proton/wiki/Requirements) and that's most likely the primary contributor to the issue you're seeing.

You're right, updating to Mint 19 almost solved the issue, I had to set up: PROTON_USE_WINED3D=1 %command% and now it works fine.

Thanks.

**Comment by antgly** [comment]
Having resolution issues on Proton Experimental and Proton 7.0-3 where at 2x in in-game settings, the screen has a black bar on top and double black bar on the bottom on my Steam Deck. In full screen mode, the screen looks vertically stretched.
